pkg/cmd/server/rhoai-normalizer/controller.go

```diff
@@ -456,11 +453,32 @@ func (r *RHOAINormalizerReconcile) innerStart(ctx context.Context, buf *bytes.Bu
 		if !ok {
 			continue
 		}
+		maa, ok2 := mas[rm.Name]
+		if !ok2 {
+			continue
+		}
 		for _, mv := range mva {
 
 			importKey, _ := util.BuildImportKeyAndURI(rm.Name, mv.Name, r.format)
 			keys = append(keys, importKey)
-			err = r.processBWriter(bwriter, buf, importKey)
+			eb := []byte{}
```

Comment:
Moving the call the CallBackstagePrinters out of LoopOverKFMR and then call it looping over the results from LoopOverKFMR, but using a new Writer each time, was the key bit in not including multiple model catalogs in one JSON entry .... with your catalog-info.YAML that data structure / indexing was as such that we could use one Writer